MAUI/TreeMap/drilldown.md

```diff
@@ -0,0 +1,391 @@
+---
+layout: post
+title: Drilldown in .NET MAUI TreeMap Control | Syncfusion
+description: Learn here all about Drilldown feature and the header customization of Syncfusion .NET MAUI TreeMap (SfTreeMap) control and more.
+platform: maui
+control: TreeMap (SfTreeMap)
+documentation: ug
+---
+
+# Drilldown in .NET MAUI TreeMap (SfTreeMap)
+
+The drilldown feature provides better visualization of hierarchy. A large set of data can be virtualized into minimal views. Each level of items can be drilled down. TreeMap provides animation along with the drilldown support. The drilldown header will be enabled when you perform drilldown in TreeMap, and this header helps in performing zoom out operation (i.e., drill up one level).
```

Comment:
The drilldown feature provides better visualization of hierarchy. A large set of data can be virtualized into minimal views. Each level of items can be drilled down. TreeMap provides animation along with the drilldown support.

Reply:
Updated the changes